src/plugins/data_views/docs/openapi/bundled.json

```diff
@@ -45,6 +45,7 @@
         "tags": [
           "data views"
         ],
+        "x-technical-preview": true,
```

Comment:
I don't think marking this as technical preview makes sense. At some point it was marked technical preview but without a clear process to remove it from technical preview and its been largely unchanged since the beginning.

Reply:
It looks like almost all the APIs in this section are marked as "technical preview": https://www.elastic.co/guide/en/kibana/master/data-views-api.html. Should I remove that from all of them?

Reply:
Wait, what content does that relate to?

I see this at the top of every api doc page

```
This documentation contains work-in-progress information for future Elastic Stack and Cloud releases. Use the version selector to view supported release docs. It also contains some Elastic Cloud serverless information. Check out our [serverless docs](https://www.elastic.co/docs/current/serverless) for more details.
```

Reply:
No, that banner is just because it's the "main"/"master" URL.  The "technical preview" indicator is this one:

![image](https://github.com/user-attachments/assets/382570cb-d643-4d39-82b1-bb405d1f4c2a)

... which seems to appear in every data view API page *except* the swap ones. Not sure if that was intentional or accidental, but whatever the right choice is, I can make it in both the OpenAPI document and the asiidoc page (until we retire the latter).

Reply:
Ah, that makes more sense! Yes, I think that can be removed.

Reply:
Thanks, I've made the change in both places and regenerated the output. It makes for a big PR, but good to get that cleaned up.